crosstool/osx_cc_configure.bzl

```diff
@@ -46,8 +46,7 @@ def _get_escaped_xcode_cxx_inc_directories(repository_ctx, xcode_toolchains):
 
     return include_dirs
 
-# TODO: Remove once Xcode 12 is the minimum supported version
-def _compile_cc_file_single_arch(repository_ctx, src_name, out_name):
```

Comment:
diff looks weird but this is deleting this function, and moving the error handling to the one below